features/admin.users.v1/components/user-profile/fields/form-field-renderer.tsx

```diff
@@ -441,7 +441,6 @@ const ProfileFormFieldRenderer: FunctionComponent<ProfileFormFieldRendererPropsI
                     name={ fieldName }
                     readOnly={ isReadOnly || isUpdating }
                     disabled={ isReadOnly || isUpdating }
-                    required={ isRequired }
```

Comment:
The required attribute is removed for the checkbox and toggle, as it does not make sense.